spec/models/generic_file_spec.rb

```diff
@@ -8,6 +8,12 @@
     @file.apply_depositor_metadata(user.user_key)
   end
 
+  it 'is initialized with blanks' do
```

Comment:
I'm not sure why this test is necessary. Seems like this is taken care of by ActiveTriples.